src/app/components/ReclaimEscrowForm/index.tsx

```diff
@@ -64,6 +69,14 @@ export const ReclaimEscrowForm = memo((props: Props) => {
     )
   }
 
+  /**
+   * `<input max="9000000000.111111111">` warns about <=9000000000.11111 so we
+   * round up to `<input max="9000000000.11112">`
+   */
+  const roundedUpStringifiedFloatMaxAmount = new BigNumber(props.maxAmount)
+    .shiftedBy(-9) // / 10 ** 9
+    .toPrecision(15, BigNumber.ROUND_UP)
```

Comment:
Chrome seems to parse `<input max="..">` into a double-float, so we need to be round up when it hits the limit of double-float precision. Is there a good way to describe this operation? And a better way to implement it?

This implementation relies on double-float having at least 15 significant decimal digits.

Reply:
I've heard other teams had to stop using `<input type="number"` because of this.